kNN_NBC/NBC/reference_strategy.h

```diff
@@ -1,4 +1,4 @@
 #pragma once
 enum ReferenceStrategy {
-	MIN_VALUE = 0, MAX_VALUE = 1
+	TI_DISABLED = -1, MIN_VALUE = 0, MAX_VALUE = 1
```

Comment:
to by ewentualnie można było przemianować na "NONE" żeby się lepiej wpasowało w kategorię określenia strategii, ale zostawiam tobie do decyzji

Reply:
@lychanl - merguj jak zdecydujesz czy to zmieniać, kod jest spoko

Reply:
Rozważałem, ale myślę że lepiej napisać explicite że się wyłącza TI jako mechanizm, a nie że nale algorytm bierze punkt bez strategii :D